app/models/solr_document.rb

```diff
@@ -49,6 +49,10 @@ def license_label
     self['license_label_ssim']
   end
 
+  def documentation
+    self['documentation_ssim']
+  end
+
```

Comment:
Which would make this redundant, since the `#solarized_methods` below already creates a method to access `documentation_tesim`